### PR TITLE
SW-4399 define and implement a clear logging mechanism during the migration stick execution

### PR DESCRIPTION
**Pull request by amrithvenkat**

### CODE REVIEW

File: src/modules/beamos/filesystem/migration/usr/bin/migrationos_to_beamos2.sh [hunk 1]
```diff
@@ -49,37 +71,73 @@ PHASE_COLOR="ORANGE"
...
+exit_code=$?
+if [ "$exit_code" -ne 0 ]; then
+  log "$(timestamp) $0: Copy syslog to the USB stick failed"
+fi
+log "$(timestamp) $0: Copying syslog to the USB stick successful"
+
+# Copy logs on the /mnt/usb to the freshly flashed SD-Card rootfs var log partition
+log "$(timestamp) $0: Copying logs to the freshly flashed SD-Card rootfs var log partition"
+log "$(timestamp) $0: System will reboot after this........"
+sudo cp -r ${LOG_LOCATION}/* ${SDCARD_ROOTFS_A_PATH}/var/log/
```
Comment (caglarkilimci) [inline src/modules/beamos/filesystem/migration/usr/bin/migrationos_to_beamos2.sh:134]: Why not we use `cmd` then the command here?

File: src/modules/beamos/filesystem/migration/usr/bin/migrationos_to_beamos2.sh [hunk 2]
```diff
@@ -49,37 +71,73 @@ PHASE_COLOR="ORANGE"
...
+sudo cp -r ${LOG_LOCATION}/* ${SDCARD_ROOTFS_B_PATH}/var/log/
+exit_code=$?
+if [ "$exit_code" -ne 0 ]; then
+  echo "$(timestamp) $0: Copying logs to the freshly flashed SD-Card rootfs var log partition failed"
```
Comment (caglarkilimci) [inline src/modules/beamos/filesystem/migration/usr/bin/migrationos_to_beamos2.sh:138]: We can use `log` to log.

**Review by caglarkilimci — CHANGES_REQUESTED** [review]

File: src/modules/beamos/filesystem/migration/usr/bin/migrationos_to_beamos2.sh (hunk 1 above)
Comment (amrithvenkat) [reply src/modules/beamos/filesystem/migration/usr/bin/migrationos_to_beamos2.sh:134]: At this stage, logs are already copied and ready to reboot.
So using cmd or log functions would not make a difference to the backed up logs

File: src/modules/beamos/filesystem/migration/usr/bin/migrationos_to_beamos2.sh (hunk 2 above)
Comment (amrithvenkat) [reply src/modules/beamos/filesystem/migration/usr/bin/migrationos_to_beamos2.sh:138]: Same as the last comment

**Review by caglarkilimci — APPROVED** [review]

**Pull request merged**